ping.py

```diff
@@ -402,10 +517,10 @@ def verbose_ping(hostname, timeout=3000, count=3,
     yield not myStats.pktsRcvd
 
 
-def quiet_ping(hostname, timeout=3000, count=3,
+def quiet_ping(hostname, timeout=3000, count=3, advanced_statistics=False,
```

Comment:
The command line uses the `verbose_ping` function which returns a simple 0 or 1.

Internally, `MStats2` always implements advanced statistics. This flag only changes the format of the tuple returned by `quiet_ping`. This makes module replacement easy for those who capture the return tuple of `quiet_ping` like this:

    maxT, minT, avgT, fracLoss = quiet_ping(...)